### ISSUE TITLE
VS insertion federation: manufacture and push MSI insertion nupkgs for 2.1/2.2

### ISSUE THREAD

**Issue by dagood** [dotnet/core-setup#7730]
Port https://github.com/dotnet/core-setup/pull/6913 to 2.1/2.2.

@dleeapho, @MichaelSimons, this tracks the work to port MSI nupkg creation back to `release/2.1` and `release/2.2`. We'll have to figure out when to make this happen.

/cc @wtgodbe

(Core-Setup tracking issue for https://github.com/dotnet/core-sdk/issues/2148 and https://github.com/dotnet/core-sdk/issues/2150.)

**Comment by dleeapho** [comment]
Waiting on the ability to test the pr.

**Comment by dagood** [comment]
Change of plan on how I plan to get publishing working, and I can test that without doing anything invasive/potentially destructive, so I can move forward: https://github.com/dotnet/core-setup/pull/7968#issuecomment-544657202.

**Comment by dagood** [comment]
The prodcon/pipebuild-side publishing infra is complete, moving this back to tracking pending tactics approval of the insertion nupkg generation infra:

release/2.1 https://github.com/dotnet/core-setup/pull/7968
release/2.2 https://github.com/dotnet/core-setup/pull/8652

**Comment by dagood** [comment]
Merged into 2.1.
Not merged into 2.2: not useful vs. 2.2 going EOL.
